go/tests/testdata/googleai-embedder-trace.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
What is the source of this trace? This does not match with what is generated by the JS SDK.

Reply:
YAY! That is exactly why we need these.
The source is the Go SDK.
Please replace with what _is_ generated by JS so the Go can match it.
Also, please implement the similar test for JS so if the JS changes, your test will break too.

Reply:
Updated the trace here -- this test now fails.